content/en/user-guide/aws/ecs/index.md

```diff
@@ -9,23 +9,211 @@ aliases:
 
+ECS tasks and services run on a Cluster.
+Execute the following command to create an ECS cluster named `mycluster`:
+
+{{< command >}}
+$ awslocal ecs create-cluster --cluster-name mycluster
+<disable-copy>
+{
+    "cluster": {
+        "clusterArn": "arn:aws:ecs:us-east-1:000000000000:cluster/mycluster",
+        "clusterName": "mycluster",
+        "status": "ACTIVE",
+        "registeredContainerInstancesCount": 0,
+        "runningTasksCount": 0,
+        "pendingTasksCount": 0,
+        "activeServicesCount": 0,
+        "settings": [
+            {
+                "name": "containerInsights",
+                "value": "disabled"
+            }
+        ]
+    }
+}
+</disable-copy>
+{{< / command >}}
+
+### Create a task definition
+
+Containers within tasks are defined by a task definition that is managed outside of the context of a cluster.
+To create a task definition that runs an `ubuntu` container forever (by running `sleep infinity` on startup), create the following file as `task_definition.json`:
+
+```json
+{
+  "containerDefinitions": [
+    {
+      "name": "server",
+      "image": "ubuntu",
+      "cpu": 10,
+      "memory": 10,
+      "command": [
+        "sleep",
+        "infinity"
+      ],
+      "essential": true
+    }
+  ],
+  "family": "myfamily"
+}
+```
+
+and then run the following command:
+
+{{< command >}}
+$ awslocal ecs register-task-definition --cli-input-json file://task_definition.json
+<disable-copy>
+{
+    "taskDefinition": {
+        "taskDefinitionArn": "arn:aws:ecs:us-east-1:000000000000:task-definition/myfamily:1",
+        "containerDefinitions": [
+            {
+                "name": "server",
+                "image": "ubuntu",
+                "cpu": 10,
+                "memory": 10,
+                "portMappings": [],
+                "essential": true,
+                "command": [
+                    "sleep",
+                    "infinity"
+                ],
+                "environment": [],
+                "mountPoints": [],
+                "volumesFrom": []
+            }
+        ],
+        "family": "myfamily",
+        "networkMode": "bridge",
+        "revision": 1,
+        "volumes": [],
+        "status": "ACTIVE",
+        "placementConstraints": [],
+        "compatibilities": [
+            "EXTERNAL",
+            "EC2"
+        ],
+        "registeredAt": 1709242097.870379
+    }
+}
+</disable-copy>
+{{< / command >}}
+
+Task definitions are immutable, and are identified by their `family` field, and calling `register-task-definition` again with the same `family` value creates a new _version_ of a task definition.
+
+### Launch a service
+
+Finally we launch an ECS service using the task definition above.
+This will create a number of containers in replica mode meaning they are distributed over the nodes of the cluster, or with the case of Fargate, over availability zones within the region of the cluster. To create a service, execute the following command:
```

Comment:
Is `with the case of Fargate` right? Honestly not sure, but it sounds weird. Maybe `in the case`?

Reply:
I think "with the case" does make sense, but "in the case" sounds better. Thanks!